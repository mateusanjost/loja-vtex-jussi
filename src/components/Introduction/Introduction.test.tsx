import TestRenderer from "react-test-renderer";
import IntroductionComponent from "./index";

describe("Introduction", () => {
  it("should match snapshot", () => {
    const testRenderer = TestRenderer.create(<IntroductionComponent />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
