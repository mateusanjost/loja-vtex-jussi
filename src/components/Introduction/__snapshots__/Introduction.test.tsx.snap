// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Introduction should match snapshot 1`] = `
<div
  className="sc-bdnxRM esbENp"
>
  <div
    className="sc-dlnjwi cITVtS"
  >
    <div
      className="sc-gtsrHT bFJHrL"
    >
      CRIAMOS LOJAS QUE VENDEM MAIS
    </div>
    <p
      className="sc-hKFxyN eiRxFJ"
    >
      A Jüssi é especialista na criação de lojas usando a plataforma VTEX. Precisa criar sua loja ou migrar de plataforma?
    </p>
    <button
      className="sc-fujyAs fgQJVB"
    >
      Veja nossas soluções
    </button>
  </div>
  <div
    className="sc-eCApnc uIFuc"
  >
    <div
      className="sc-gKAaRy Qhblr"
    >
      <div
        className="sc-iCoGMd cobULX"
      >
        <button
          className="sc-fujyAs cTXVKe"
        >
          Mais detalhes
        </button>
      </div>
    </div>
    <div
      className="sc-jSFjdj fNMQKG"
    >
      <div
        className="sc-iCoGMd dwofpY"
      >
        <button
          className="sc-fujyAs cTXVKe"
        >
          Comprar em 12x
        </button>
      </div>
      <div
        className="sc-iCoGMd enjqMK"
      >
        <button
          className="sc-fujyAs cTXVKe"
        >
          Adicionar à sacola
        </button>
      </div>
    </div>
  </div>
</div>
`;
